### PR TITLE
FEATURE: open create invite modal from the bootstrap notice banner

### PR DESCRIPTION
**Pull request by techAPJ**
https://user-images.githubusercontent.com/5732281/185048711-2b232215-f92e-4359-b639-c65375108cd8.mov

### CODE REVIEW

File: app/assets/javascripts/discourse/app/templates/components/bootstrap-mode-notice.hbs [hunk 1]
```diff
@@ -4,7 +4,7 @@
       {{this.message}}
     </div>
     <div class="col-button">
-      <div class="alert--button"><DButton @icon="user-plus" @href="/my/invited" @class="btn-primary bootstrap-invite-button" @label="bootstrap_invite_button_title" /></div>
+      <div class="alert--button"><DButton @icon="user-plus" @action="inviteUsers" @class="btn-primary bootstrap-invite-button" @label="bootstrap_invite_button_title" /></div>
```
Comment (tgxworld) [inline app/assets/javascripts/discourse/app/templates/components/bootstrap-mode-notice.hbs:7]:
```suggestion
      <div class="alert--button"><DButton @icon="user-plus" @action={{this.inviteUsers}} @class="btn-primary bootstrap-invite-button" @label="bootstrap_invite_button_title" /></div>
```

It seems like passing a string in is deprecated and a warning is logged in the console if we do this.

**Review by tgxworld — APPROVED** [review]
One small comment but otherwise changes look good :+1:

File: app/assets/javascripts/discourse/app/templates/components/bootstrap-mode-notice.hbs (hunk 1 above)
Comment (techAPJ) [reply app/assets/javascripts/discourse/app/templates/components/bootstrap-mode-notice.hbs]: Thanks for the suggestion, made the change!

**Review by Ellajoke — COMMENTED** [review]
Get me Approved General

**Pull request merged**